config/default/core.entity_view_display.taxonomy_term.blog_type.default.yml

```diff
@@ -18,26 +18,10 @@ content:
     label: hidden
     settings: {  }
     third_party_settings: {  }
-    weight: 1
```

Comment:
Do we need these changes?

Reply:
yep https://github.com/ytgov/yukon-ca/pull/51/files#diff-61d992d93f6cf9a2559a7d5f7e890a682d361e6d9c305276eef08e96ea8107daR21

Reply:
reverted